### DISCUSSION TITLE
Can't update my wesite anymore

### DISCUSSION THREAD

**Discussion by fenqiling** [alshedivat/al-folio#2262] [Q&A]
I tried to update the local files (publication) of my website and then push it to github, but it failed and then changed all my local files to some random files. Please tell me how I can fix this issue. I have attached my website and github repo link here. Thank you in advance!

https://www.fenqiw.com/
https://github.com/fenqiling/fenqiling.github.io

**Comment by george-gca** [comment]
Which branch were you on? Git doesn't change local files unless you tell it to. Can you provide more information?

**Reply by fenqiling** [reply]
I'm trying to update it on gh-page. I'm not a expert on this. Could you please help me check into this issue?

**Reply by george-gca** [reply]
You should never touch the `gh-pages` branch. All the changes you make should be made on the main branch or any other branch. The `gh-pages` branch is automatically overwritten every time you make a change in the main branch.

**Reply by fenqiling** [reply]
I think I didn't touch the gh-pages. I just did as I did before, but it's maybe like you said. Is there solutions for this? Do I have to build the website from the scratch?

**Reply by george-gca** [reply]
Since you are not sure of what you did, I believe the easiest solution is to start fresh. I would suggest you put aside your current project, start fresh, then compare both directories using tools like [meld](https://meldmerge.org/) or [winmerge](https://winmerge.org/).
